src/python/pants/engine/internals/mapper.py

```diff
@@ -154,7 +154,6 @@ class AddressMapper:
     prelude_glob_patterns: Tuple[str, ...]
     build_patterns: Tuple[str, ...]
     build_ignore_patterns: Tuple[str, ...]
-    subproject_roots: Tuple[str, ...]
```

Comment:
This option was being accessed by looking at `GlobalOptions`, rather than `AddressMapper`. That works well for us because we can now move that option out of the bootstrap options, and it makes it easier for us to delete `AddressMapper`.